sos/report/plugins/memory.py

```diff
@@ -26,12 +26,15 @@ def setup(self):
             "/proc/slabinfo",
             "/proc/pagetypeinfo",
             "/proc/vmallocinfo",
+            "/sys/devices/system/memory",
```

Comment:
Note that on bigger system with memory in TBs this will increase the data collection time drastically. 
If we really need this data then consider making this as optional data collection.

Reply:
Thanks Vasant,
will update your comments